proposed/2022/DotnetUpdatePackage.md

```diff
@@ -0,0 +1,179 @@
+```
+
+- Displays what would be updated in the update operation for the current working directory.
+
+```
+dotnet update package --dry-run
+```
+
+#### Exit Codes
+
+- 0 - NuGet succeeded in update.
+- 1 - NuGet failed to update.
+
+### Technical explanation
+
+<!-- Explain the proposal in sufficient detail with implementation details, interaction models, and clarification of corner cases. -->
+
+## Drawbacks
+
+<!-- Why should we not do this? -->
+
+While `dotnet add package` provides some of this functionality, it doesn't give much control, ease-of-use, or discoverability to a simple update command.
+
+## Rationale and alternatives
+
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+
+There currently doesn't exist a .NET CLI equivalent of this command and the impact of not doing this can lead to small papercuts, general dissatisfaction compared to other package management experiences and extensive workarounds instead of a common command that provides a solution.
+
+While there has been one design in the past, it was not extensive enough to gather feedback from the community and surrounding teams on whether the direction is sound to pursue.
+
+## Prior Art
+
+<!-- What prior art, both good and bad are related to this proposal? -->
+<!-- Do other features exist in other ecosystems and what experience have their community had? -->
+<!-- What lessons from other communities can we learn from? -->
+<!-- Are there any resources that are relevant to this proposal? -->
+
+- [NuGet update](https://docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-update)
+- [cargo update](https://doc.rust-lang.org/cargo/commands/cargo-update.html)
+- [npm update](https://docs.npmjs.com/cli/v8/commands/npm-update)
+- [RubyGems update](https://rubygems.org/gems/rubygems-update)
+- [dart pub upgrade](https://dart.dev/tools/pub/cmd/pub-upgrade)
+- [pip install -U](https://pip.pypa.io/en/stable/cli/pip_install/)
+
+## Unresolved Questions
+
+<!-- What parts of the proposal do you expect to resolve before this gets accepted? -->
+<!-- What parts of the proposal need to be resolved before the proposal is stabilized? -->
+<!-- What related issues would you consider out of scope for this proposal but can be addressed in the future? -->
+- Should this command use `dotnet add package` under the hood for the scenarios that make sense such as finding the latest version of a specific package name?
+- What verb makes most sense? `update`, `upgrade`, `refresh`, etc? Update seems the most consistent for package managers such as cargo, npm, rubygems, and NuGet. Upgrade seems the most consistent for package managers such as pip.
+- Should `--highest-minor` and `--highest-patch` be replaced with a single `--dependency-version <version>` parameter to account for more scenarios like `Lowest(default)` and `Highest`?
```

Comment:
The `Lowest` vs `Highest` is a larger feature than the update command itself. 
it makes it difficult for me to suggest a name based on what that feature may turn out to be.